crt_portal/cts_forms/views.py

```diff
@@ -26,7 +27,7 @@ def IndexView(request):
 
     # Sort data based on request from params, default to `created_date` of complaint
     sort = request.GET.getlist('sort', ['-create_date'])
-    per_page = request.GET.get('per_page', 15)
+    per_page = request.GET.get('per_page', 1)
```

Comment:
The default should be 15

Reply:
Checked locally and that will fix all the tests. I assume you would want me to make the change. I will confirm with you before merging.

Reply:
Oh I can just change it back, I just needed it set to one to check the 'end of results' message logic!